src/components/ArrowButton.js

```diff
@@ -69,12 +69,13 @@ ArrowButton.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]).isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
I'm thinking a  button requires an onClick function every time? Is there a time when it doesn't?